source/connect.txt

```diff
@@ -122,6 +122,57 @@ option for programmatic usage of ``mongosh``, like a :driver:`driver
    - To provision access to a MongoDB deployment, see :manual:`Database
      Users </core/security-users/>`.
 
+
+Connect with LDAP
+~~~~~~~~~~~~~~~~~
+
+When using LDAP for authorization, users connecting via
+``mongosh`` must:
+
+- Set :option:`--authenticationMechanism <mongosh --authenticationMechanism>` to the appropriate authentication
+  mechanism.
+
+  If using :ref:`LDAP authentication <security-ldap>`, set this to ``PLAIN``.
+
+  If using :ref:`Kerberos authentication <security-kerberos>`, set this to
+  ``GSSAPI``.
+
+  If using :ref:`x.509 <security-auth-x509>`, set this to ``MONGODB-X.509``.
```

Comment:
```suggestion
  If using :ref:`x.509 <security-auth-x509>`, set this to ``MONGODB-X509``.
```

That being said … why are Kerberos & X.509 listed here? Not really wrong, but I wouldn’t expect information like this under a “Connect with LDAP” section.

Reply:
Good point, I removed those two options and cleaned up the language to be more focused specifically on LDAP auth. Thanks for pointing this out!